docs/mobile-apps/virtual-usb.md

```diff
@@ -15,164 +15,247 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
+  As a reminder, vUSB only works on private devices (marked with a <img src={useBaseUrl('img/virtual-usb/vusb-private.jpg')} alt="Sauce Labs Private Device Icon" width="15" />). A quick way to find your organization's private devices from the device list is to click **Filters** and toggle **Private Devices**.
 
-<img src={useBaseUrl('img/virtual-usb/vusb-privatedevices.jpg')} alt="Virtual USB Private Devices Filters" width="400" />
+  <img src={useBaseUrl('img/virtual-usb/vusb-privatedevices.jpg')} alt="Virtual USB Private Devices Filters" width="500" />
 
-5. Grab your Sauce Labs credentials (username and access key), which you can find under **Account** > **User settings**.
-6. Open a second command line terminal window, while keeping the first window running in the background. In the new window, run this command line to launch your vUSB session, authenticate your credentials (`username` and `accessKey`), and connect to your desired device:
+### Connect to Server
 
-```java
-java -jar vusb-client.jar startSession --username john.smith --accessKey ab015c1e-1997-4794-8g52-32f7fe110e03 --deviceName Motorola_Moto_Z_real
-```
-Your chosen device will be allocated specifically for your test session. It will not available for other users until your session is over.
+4. On your local machine, launch a command line terminal window and use `cd` to navigate to the folder where you downloaded the vUSB client.
+
+5. Enter the `server` command, followed by the `--datacenter US` (Sauce Labs U.S. Data Center) or `--datacenter EU` (Sauce Labs Europe Data Center). This establishes the connection from your local machine to our Real Device Cloud, where your private devices are hosted.
+  ```java
+  java -jar virtual-usb-client-2.0.0.jar server --datacenter US
+  ```
 
-7. When vUSB connects to your live testing session, it will return a success message along with a link you can click to watch the device running your test in real time. Here's an example of a success message:
+  **Set Up a Local Proxy (Optional)**
 
+  You can launch a proxy tunnel or device proxy tunnel using the proxy command options (run `java -jar virtual-usb-client-2.0.0.jar --help` and go to the `server` section).
+  <img src={useBaseUrl('img/virtual-usb/vusb-proxy.jpg')} alt="Virtual USB Proxy Command Options" width="500" />
+
+  **Set Environment Variables (Optional)**
+
+  You can set your Sauce Labs user credentials (`username` and `accessKey`) as environment variables. For more info, see [Using Environment Variables for Authentication Credentials](https://wiki.saucelabs.com/pages/viewpage.action?pageId=48365647#BestPracticesforRunningTests-UseEnvironmentVariablesforAuthenticationCredentials).
+
+### Start Test Session
+
+6. Choose one of the options below to establish the connection to your device and start your vUSB test session.
+
+  **Option 1: No Proxy (default)**
```

Comment:
I think you mean `No Sauce Connect Tunnel` here. 
My advice would be to:

- remove the "No Proxy" and just call it `default`. => `Option 1: Default`
- have a second option => `Option 2: Use Sauce Connect Proxy`
- have a third option => `Option 3: Use a device proxy` with options like `--proxyHost  --proxyPassword --proxyPort  --proxyUser`

Reply:
Question about the device proxy! I noticed overlapping flags that appear under both the **Start vUSB-Server** (step 5 in doc) and **startSession** (step 6 in doc). Does this mean users can configure their device proxy in either step?

    **server      Start vUSB-Server**
      Usage: server [options]
        Options:
          --serverHost
            vUSB-Server host
            Default: http://127.0.0.1
          --serverPort
            vUSB-Server port
            Default: 33657
          --adbPortMin
            vUSB-Server adb port
            Default: 7000
          --adbPortRange
            vUSB-Server adb port range
            Default: 100
        * --datacenter
            Data center where your tests will run
            Possible Values: [CUSTOM, EU, US]
          --proxyHost
            HTTP proxy host
          --proxyPort
            HTTP proxy port
            Default: 0
          --proxyUser
            HTTP proxy user
          --proxyPassword
            HTTP proxy password
          --startUsbmuxd
            Starts the usbmuxd server immediately
            Default: false




    **startSession      Start a new session
      Usage: startSession [options]
        Options:
        * --username
            The username.
        * --accessKey
            The access key.
          --serverHost
            vUSB-Server host
            Default: http://127.0.0.1
          --serverPort
            vUSB-Server port
            Default: 33657
        * --deviceName
            Specify a device descriptor for static allocation
          --proxyHost
            Specify a proxy host to be set on the device
          --proxyPassword
            Specify a proxy password to be set on the device
          --proxyPort
            Specify a proxy port to be set on the device
            Default: 0
          --proxyUser
            Specify a proxy user to be set on the device
          --tunnelIdentifier
            Specify a Tunnel Identifier for SauceConnect

Reply:
@kimsauce 

No, a server might need to have a proxy, because it communicates over that proxy in terminal 1, and a device needs to have a proxy, because the proxy is used to start a connection with the device, which means both are needed

Reply:
ahh i see-- thank you for clarifying

Reply:
YW